python/mxnet/gluon/trainer.py

```diff
@@ -113,6 +113,36 @@ def _init_kvstore(self):
 
         self._kv_initialized = True
 
+
+    @property
+    def learning_rate(self):
```

Comment:
document this as 
```Properties
----------```

in init doc

Reply:
Also report learning_rate when using lr_sheduler

Reply:
resolved